src/app.js

```diff
@@ -0,0 +1,52 @@
+import express from 'express';
+import { urlencoded, json } from 'body-parser';
+import cors from 'cors';
+import router from './routes/index';
+import ErrorHandler from './utils/error';
+import logger from './utils/winston';
+import Responses from './utils/response';
+import config from './config';
+
+const isDevelopment = config.env;
+
+const app = express();
+
+app.use(cors());
+
+app.use(urlencoded({ extended: false }));
+app.use(json());
+
+// used in testing heroku
+app.get('/', (req, res) => Responses.handleSuccess(200, 'Welcome to Barefoot Nomad', res));
+
+// to throw an error to the logger
+// eslint-disable-next-line no-unused-vars
+app.get('/err', (req, res) => {
```

Comment:
What is this middleware for?

Reply:
I was testing if `throw new error` works

Reply:
@timolinn  should I remove it

Reply:
Yes please, since there's no other use for it. Its #DeadCode.

Reply:
okaay thanks.